locale/en_US/locale.po

```diff
@@ -22,37 +22,16 @@ msgstr ""
 "This plugin enables customization of the default user interface text used by "
 "OJS and OMP."
 
-msgid "plugins.generic.customLocale.search.fileName"
-msgstr "Search in file names"
-
-msgid "plugins.generic.customLocale.search.key"
-msgstr "Search for key"
-
 msgid "plugins.generic.customLocale.saveAndContinue"
 msgstr "Save and continue"
 
 msgid "plugins.generic.customLocale.noneCreated"
-msgstr "No files found."
-
-msgid "plugins.generic.customLocale.printChanges"
-msgstr "Download documentation"
-
-msgid "plugins.generic.customLocale.localeKey.description"
-msgstr "Enter an entire locale key to search for."
-
-msgid "plugins.generic.customLocale.path"
-msgstr "Action"
-
-msgid "plugins.generic.customLocale.localeKey"
-msgstr "Key"
-
-msgid "plugins.generic.customLocale.localeKeyValue"
-msgstr "Value"
+msgstr "No locales found."
 
 msgid "plugins.generic.customLocale.customize"
 msgstr "Customize"
 
-msgid "grid.action.printChanges"
+msgid "plugins.generic.customLocale.printChanges"
 msgstr "Download a file that lists all the customizations you made."
```

Comment:
This looks weird in the plugin list:
![image](https://user-images.githubusercontent.com/200411/160497713-951af129-0864-4994-96ef-1e64e1d5636d.png)

I would recommend "Download changes" or something brief like that.